RELEASE.md

```diff
@@ -1,98 +1,64 @@
 # Releases
 
-There are three types of Renku releases: Sprint release, Hot/bug fix release and Quick release.
+There are two types of Renku releases: planned and unplanned.
 
-## Sprint release
+A release consists of: 
 
-A sprint release is planned on the Tuesday after the end of every sprint
-(every three weeks).
-The current state of the main branch is released as is, only checking the
-wording in the changelog to ensure it is of good quality and adding the version
-information.
+* Helm chart changes to reflect the new versions of individual components
+* Changes to `CHANGELOG.rst`
+* A new tag in this repository, which results in packaging and pushing out new helm charts and corresponding docker images
 
-### Update components [Graph, UI, RP]
+This procedure should be followed for *any* release:
 
-🧑‍🏭 Component maintainers can tag a release in their local repositories whenever
-they want, the schedule is up to them. They are responsible for updating the
-`CHANGELOG.rst` file in the auto-PR in the Renku repository.
-Please create a new section above the last release if no one has already done it.
-Please follow the `CHANGELOG-template.rst` for formatting and ordering the changelog.
-The auto-PRs **must** be merged with an updated changelog/release notes. If upgrading
-a service will knowingly result in an extended outage (e.g. because of a DB migration)
-this *must* be clearly noted and highlighted in the release notes. Use emojis freely.
-Make sure YAT is aware of any expected complications and plan to be involved in
-the rollout process. YAT and the product team have to approve those PRs before they
-can be merged, so changes and the changelog are double-checked.
+* create a release branch (e.g. `0.46.x`) if one does not already exist

```

Comment:
mention/link to the create release pr action https://github.com/SwissDataScienceCenter/renku/actions/workflows/create-release-branch.yml which also creates the changelog entry. This is merged and available

Reply:
Done.